hack/build-images.sh

```diff
@@ -125,7 +122,7 @@ function image {
 ( image openshift/origin-docker-registry       images/dockerregistry ) &
 ( image openshift/origin-egress-router         images/router/egress ) &
```

Comment:
Don't see any reason to overlap them.  And no, they should be image because we are capturing output.